.github/workflows/build-docs.yml

```diff
@@ -24,13 +24,10 @@ jobs:
       - uses: mamba-org/setup-micromamba@v2.0.4
         with:
           environment-file: ./environment.yml
-      - name: Check dependencies
-        run: |
-          micromamba activate ultraplot-dev
-          micromamba list
```

Comment:
You needed `shell: bash -el {0}` in this step. The eval command you added below is not needed.